compiler/llvm/clangUtil.cpp

```diff
@@ -3854,6 +3878,19 @@ void makeBinaryLLVM(void) {
     clangLDArgs = runtimeArgs;
   }
 
+  // Grab extra dependencies for multilocale libraries if needed.
+  if (fMultiLocaleInterop) {
+
+    // TODO: Has to be a more friendly way to do this?
+    clangLDArgs.push_back("-L/usr/local/lib");
```

Comment:
This doesn't seem right. What `-L` options are you looking for here? Is this something that the compileline command below should get? If not maybe it needs updating?

Reply:
I would like a system independent way of "always search the system and user libraries". I don't think that the built in clang has those search paths. It has been awhile though (and I'm aware this is OS specific). I will try removing and see if any runs break.

Reply:
Removed it.